src/client/ui/styles.less

```diff
@@ -200,6 +200,7 @@
         top: 0;
         width: 35px;
         height: 57px;
+        pointer-events: none;
```

Comment:
Can we use the same solution for the `click` action to reduce blinking of the botton bar?